package.json

```diff
@@ -12,7 +11,10 @@
     "prebuild": "node ./scripts/update-endpoints.js",
     "build": "docusaurus build",
     "swizzle": "docusaurus swizzle",
-    "predeploy": "node ./scripts/update-endpoints.js",
+    "update-endpoints": "node ./scripts/update-endpoints.js",
+    "update-cli-help": "node ./scripts/update-cli-help.js",
+    "update-evm-config-schema": "curl https://raw.githubusercontent.com/enviodev/hyperindex/main/codegenerator/cli/npm/envio/evm.schema.json > ./static/schemas/config.evm.json",
+    "predeploy": "yarn update-endpoints && yarn update-cli-help && yarn update-evm-config-schema",
```

Comment:
Let's use `npm run` so it doesn't require installing `yarn`

Reply:
This whole project is setup with yarn, its a bit of a pain. I want to change it out.

Reply:
If you look a couple lines down there is even a preinstall hook to `npx only-allow yarn`

Think we can migrate soon.